openfifth/aspen-discovery
# Explore more bar

TEST PLAN:
1. Carry out a search, it can be a library catalogue search or a Summon Search. 
2. You should see that under the second or third result you have an Explore More box (this sometimes does not appear in a catalogue search depending on your search term). 
3.  Apply the patch. 
4. Navigate to Library Systems and select your Library. 
5. Scroll down to the 'Explore More Bar Section'.
6. You will see options to control whether the Explore More Bar displays when carrying out a range of different searches. 
7. Untick the option for Catalogue or Summon search and repeat step 1. 
8. There should no longer be an Explore More Bar in your results.
